hwdb.d/20-ieee1394-unit-vendor-model.hwdb

```diff
@@ -28,3 +28,257 @@
 #
 # Additionally, VENDOR_ID_FROM_DATABASE and MODEL_ID_FROM_DATABASE environment variables are
 # preferable.
+
+#
+# BridgeCo. Enhancement BreakOut Box (BeBoB) for DM1000, DM1100, and DM1500 ASICs.
+#
+
+ieee1394:node:ven0x000aacmo0x000002units0x00a02d:0x010001
+ieee1394:ven00000AACmo00000002sp0000A02Dver00010001
+  VENDOR_ID_FROM_DATABASE=Acoustic Reality
+  MODEL_ID_FROM_DATABASE=eAR FireWire Audio
+  FW_UNIT_FUNC=audio_and_music
```

Comment:
not a fan of needless abbreviations. Maybe call this FIREWIRE_UNIT_FUNCTION?

Reply:
also, wouldn't this be better as separate fields?

i.e. FIREWIRE_UNIT_FUNCTION_AUDIO=1 + FIREWIRE_UNIT_FUNCTION_VIDEO=1 + FIREWIRE_UNIT_FUNCTION_MUSIC=1?

i mean, i have no idea about firewire, and what this precsiely means, I am just wondering...  I mean, in the input stuff we do it like that: we have a couple of bools that identify a device as mouse, keyboard, trackpad, touchpad, …, this feels similar, … or not?